adder.go

```diff
@@ -7,3 +7,7 @@ func Add(n1, n2 int) int {
 func Multiply(n1, n2 int) int {
 	return n1 * n2
 }
+
+func Divide(dividend, divisor int) int {
+	return dividend / divisor
```

Comment:
Are you sure there are _zero_ problems in this code? Do we want to get to Einstein?